mods/ts/rules/gdi-vehicles.yaml

```diff
@@ -12,6 +12,8 @@ APC:
 	Mobile:
 		ROT: 5
 		Speed: 113
+		TerrainSpeeds:
+			Water: 70
```

Comment:
Can someone with a working knowledge of Tiberian Sun unit stats (or with TibEd and Tiberian Sun installed) confirm that this value is correct please?
(@reaperrr, @abcdefg30)